cmd/rook/agent/agent.go

```diff
@@ -13,52 +13,59 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package main
+package agent
```

Comment:
what if we kept agent.go and rook.go in the main package? Then only backend-specific commands would need a subpackage